### PR TITLE
Feature/0 cleanup boilerplate code and making project structure

### PR DESCRIPTION
**Pull request by katamatata**
### Description
I have cleaned boilerplate code from unnecessary files and created mockups for Header and HomepageImage components. I also have added reset.css file and basic styling to these components using a styled-components approach.

### How it was done
Don't forget to `yarn install` first to install missing dependencies.

There is a components folder where all the components stored. Each component is stored in its own folder. In each component folder, there is a file `ComponentNameElements.jsx` that contains styling for this particular component. (I have a problem with Prettier/ESLint formatter, so I have renamed all `.js` files to `.jsx` until I find how to fix formatting issue).

There is also a common folder for components that we would reuse multiple times in our app.

### CODE REVIEW

File: src/components/Header/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+import { ContentWrapper } from '../../common';
+import { HeaderWrapper, Logo } from './HeaderElements';
+
+export const Header = () => (
+  <HeaderWrapper>
+   <ContentWrapper>
+    <Logo src='./logo.png' alt="Logo" />
```
Comment (helloanil) [inline src/components/Header/Header.jsx:9]: It's a better practice do it like following:

```js
import logoImg from './logo.png';

<Logo src={logoImg} alt="Logo" />
```

You can even do this in the Logo component itself.

File: src/components/HomepageImage/HomepageImageElements.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+
+export const ImageWrapper = styled.div`
+  background-color: orange;
+  padding: 120px 0;
+`;
+
+// export const TextWrapper = styled.div`
+
+// `;
```
Comment (helloanil) [inline src/components/HomepageImage/HomepageImageElements.js:10]: Leaving commented code is always a no-no. You can remove this code if you don't need it.

File: src/components/Header/Header.jsx (hunk 1 above)
Comment (katamatata) [reply src/components/Header/Header.jsx:9]: I tried this way and it's not working, so I googled another way.

image

File: src/components/HomepageImage/HomepageImageElements.js (hunk 2 above)
Comment (katamatata) [reply src/components/HomepageImage/HomepageImageElements.js:10]: Thanks, I'll fix it.

**Pull request merged**